cookbooks/composition-active-tracers.prm

```diff
@@ -105,6 +105,7 @@ subsection Postprocess
     set Data output format = vtu 
     set List of tracer properties = velocity, initial composition
     set Interpolation scheme = cell average
+    set Exchange ghost particles = true
```

Comment:
The error message in #1376 says "Update ghost particles" though, is that error message incorrect?